### DISCUSSION TITLE
Trouble with running ElastAlert from Docker

### DISCUSSION THREAD

**Discussion by roman-tasi** [jertel/elastalert2#600] [Q&A] [answered]
Hi, I am new to ElastAlert with Docker. I ran this step:

`docker run -d -v /tmp/elastalert.yaml:/opt/elastalert/config.yaml ghcr.io/jertel/elastalert2/elastalert2`

and created the container in my Centos7 VM. But how do I actually start running ElastAlert commands such as `elastalert-create-index` in the docker container?

**Comment by jertel** [comment, marked as answer]
You don't need to run any commands in the container. Once you start the container it automatically runs `elastalert-create-index` and then starts the `elastalert` process. At that point ElastAlert2 is running and monitoring your rules.
